lib/Unexpected.js

```diff
@@ -488,6 +488,8 @@ Unexpected.prototype.fail = function (arg) {
             var value = arg[key];
             if (key === 'diff') {
                 error.createDiff = value;
+            } else if (key === 'stack') {
+                error._stack = value;
```

Comment:
I'm fine with just having the stack - but I don't really like the half secret `_stack` field. I think we should make this a first class API using expect.fail.